app/nav/CmsNavElement.tsx

```diff
@@ -0,0 +1,38 @@
+import type { IconType } from "react-icons";
+import { Center, Flex, Icon, Text, useColorModeValue } from "@chakra-ui/react";
+import { Link } from "~app/links";
+
+export default function NavElement({
+	href,
+	name,
+	icon,
+}: {
+	href: string;
+	name: string;
+	icon: IconType;
+}) {
+	const textColor = useColorModeValue("gray.700", "gray.400");
```

Comment:
I think there's quite a bit of over-reliance on manually setting colors via `useColorModeValue` here. When used correctly, chakra-ui sets colors appropriately, according to the theme, most of the time by default. Compare your current approach with the existing `Nav.tsx` component for example (which uses a single `useColorModeValue` in the entire tree just to enable the background blur effect on browsers that support it).

Reply:
Ok, i will have a look

Reply:
Whenever you define "custom" colors (e.g darker background when hovering) it is unavoidable to use `useColorModeValue` because chakra wont automticially change the color to fit dark/light mode.

Reply:
Why define custom colors then? If the defaults really don't fit (which I'd doubt from what I've seen), then we should modify the theme instead to apply those color changes uniformly.

Reply:
How would you implement different background colors without using the `useColorModeValue` hook?